src/command/pinning/pin.ts

```diff
@@ -18,8 +18,11 @@ export class Pin extends RootCommand implements LeafCommand {
       await this.bee.pinCollection(this.address)
       this.console.log('Pinned successfully')
     } catch (error) {
-      if (error.message === 'Internal Server Error') {
-        this.console.error('Could not pin ' + this.address)
+      this.console.error('Could not pin ' + this.address)
+
+      if (error.message === 'Not Found') {
+        this.console.error('No root chunks found with that address.')
```

Comment:
```suggestion
        this.console.error('No root chunk found with that address.')
```

Reply:
Thanks, I agree it reads better that way.